includes/admin/class-evf-admin-import-export.php

```diff
@@ -0,0 +1,163 @@
+<?php
+/**
+ * EverestForms Import Export Class
+ *
+ * @package EverestForms\Admin
+ * @since   1.5.11
```

Comment:
Inconsistent version use `1.6.0`

Reply:
The versioning instructed has been implemented.